src/frontend/js/modules/previous-page.js

```diff
@@ -11,12 +11,13 @@ angular.module('esn.previous-page', [])
         // TODO: Write tests for the newly changed logic: https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/12, https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/74
```

Comment:
Edit the comment so that we write tests for these later :trollface:

Reply:
create an issue for that